src/js/src/components/SearchResult.vue

```diff
@@ -82,6 +118,15 @@ export default {
       this.requestFacets({query:this.query, facets:this.searchAppliedFacets, options:this.solrSettings})
       this.$_pushSearchHistory('SolrWayback', this.query, this.searchAppliedFacets, this.solrSettings)
     },
+    getOffsetArray(array, number) {
+      let newArray = [...array]
```

Comment:
Nice use of spread operator here  - good for simple Arrays (and other iterable objects for that matter) 👍

Reply:
\o/